platformio.ini

```
@@ -0,0 +1,27 @@
+; PlatformIO Project Configuration File
+;
+;   Build options: build flags, source filter
+;   Upload options: custom upload port, speed and extra flags
+;   Library options: dependencies, extra library storages
+;   Advanced options: extra scripting
+;
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/page/projectconf.html
+
+[platformio]
+src_dir = OpenBikeSensorFirmware
+
+[env:esp32dev]
+platform = espressif32
+board = esp32dev
+framework = arduino
```

```suggestion
framework = arduino
monitor_speed = 115200
```
Does this superseeds the .idea/baut_rate settings?

---

Die Einstellungen bedeuten fast das gleiche, aber gehören zu zwei verschiedenen Tools:
- CLion's Serial Monitor
- PlatformIO's Serial Monitor